config/sync/shortcut.set.default.yml

```diff
@@ -1,4 +1,4 @@
-uuid: null
+uuid: 7880aeb5-f773-4a13-a709-19acbdf9b719
```

Comment:
Reminder to avoid committing changes to this file. I caught it this time around so it didn't cause any problems, but otherwise it would break config imports on a fresh install.